x-pack/plugins/reporting/server/export_types/csv/generate_csv/index.ts

```diff
@@ -37,7 +37,7 @@ interface SearchRequest {
 }
 
 export interface GenerateCsvParams {
-  browserTimezone: string;
+  browserTimezone: string | undefined;
```

Comment:
If time zone is not present (aka an old POST URL has been copied before the params were set to include timezone), then the Advanced Settings will be sourced. If that is `Browser`, then UTC will be used.